commit 1655cb3baa5b536a5204df6e6c416bb39c161c26
Author: Fabio Erculiani <lxnay@sabayon.org>
Date:   Tue Feb 12 14:22:56 2013 +0100

    [RigoDaemon] drop send_requested_reply from dbus config, causes deadlock with gdm-3.6

diff --git a/rigo/RigoDaemon/dbus/org.sabayon.Rigo.conf b/rigo/RigoDaemon/dbus/org.sabayon.Rigo.conf
index 364c901..64d3e84 100644
--- a/rigo/RigoDaemon/dbus/org.sabayon.Rigo.conf
+++ b/rigo/RigoDaemon/dbus/org.sabayon.Rigo.conf
@@ -5,15 +5,12 @@
         <policy user="root">
                 <allow own="org.sabayon.Rigo"/>
                 <allow send_destination="org.sabayon.Rigo"/>
-		<allow send_requested_reply="true"/>
         </policy>
         <policy group="entropy">
                 <allow own="org.sabayon.Rigo"/>
                 <allow send_destination="org.sabayon.Rigo"/>
-		<allow send_requested_reply="true"/>
         </policy>
         <policy context="default">
-		<allow send_requested_reply="true"/>
 		<deny send_destination="org.sabayon.Rigo"/>
         </policy>
 </busconfig>
